content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -354,7 +354,14 @@ The expected values are:
 - `scope`: the channel code for which the value is assigned (optional)
 - `format`: format of the date following the PHP format [specification](https://www.php.net/manual/en/function.date.php) (optional, only relevant for `date` attributes). By default, it is *Y-m-d* (e.g. *2020-01-31*)
 - `currency`: the currency code for which the price is assigned (optional, only relevant for `price_collection` attributes). By default, all the prices in the collection are displayed, separated by a comma.
-- `label_locale`: the locale code for the label of the option or record (optional, only relevant for `simple select`, `multi-select`, `reference entity single link` and `reference entity multiple link` attributes). By default, the code of the option is used. Here is an example:
+- `label_locale`: the locale code for the label of the option or record (optional, only relevant for `simple select`, `multi-select`, `reference entity single link` and `reference entity multiple link` attributes). By default, the code of the option is used.
+- `unit_label_locale`: the locale code for the unit of the measurement (optional, only relevant for `measurement` attributes). By default, the code of the unit is used.
+
+:::warning
+We don't manage grammar corrections when using a measurement attribute type in a concatenation (i.e: no plural management).
```

Comment:
I don't really understand the message here

Reply:
which one? 
This new option (unit_label_locale) is useful if the client wants to concatenate a measurement attribute type. Before, we used the unit code instead of the unit label. It means that the result of the concatenation could be: "il pèse 100 Gram" because "Gram" was the unit code. Now, if "Gram" is translated in french, the result could be "il pèse 100 Gramme". 
I added the warning mention because as you can see in my example, we are not able to manage plurals for instance.